docs/src/pages/system/basics/basics.md

```diff
@@ -1,338 +1,215 @@
-# @material-ui/system
+# Material-UI System
 
-<p class="description">Styled system & style functions for building powerful design systems.</p>
+<p class="description">Utility-first style functions for rapidly building custom designs & systems.</p>
 
-## Getting Started
+The system lets you quickly build custom UI components leverating the design tokens defined in your theme.
 
-`@material-ui/system` provides low-level utility functions called "_style functions_" for building powerful design systems. Some of the key features:
+## Demo
 
-- ⚛️ Access the theme values directly from the component props.
-- 🦋 Encourage UI consistency.
-- 🌈 Write responsive style effortlessly.
-- 🦎 Work with any theme object.
-- 💅 Work with the most popular CSS-in-JS solutions.
-- 📦 Less than [4 KB gzipped](https://bundlephobia.com/result?p=@material-ui/system).
-- 🚀 [Fast enough](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-benchmark/README.md#material-uisystem) not to be a bottleneck at runtime.
+{{"demo": "pages/system/basics/Demo.js"}}
 
-It's important to understand that this package exposes pure (side-effect free) style functions with this signature: `({ theme, ...style }) => style`, **that's it**.
+## Why
 
-### Demo
+There are several reasons for why you may need the system offered by Material-UI. Here are few of them:
 
-In the rest of this _Getting Started_ section we are using **styled-components** as the reference example (to emphasize the universality of this package). Alternatively, you can [emotion](#interoperability) or any other CSS-in-JS styling solution.
-The demos are also based on the **default** Material-UI [theme object](/customization/default-theme/).
+### 1. Building consistent UIs is hard
 
-{{"demo": "pages/system/basics/Demo.js", "defaultCodeOpen": true}}
+This is especially true, when there is at least more than one person building the application. There has to be some synchronization to what are the design tokens and how are those used. What parts of the theme structure should be used with what CSS properties etc.
 
-### Installation
+### 2. Switching context between JS and CSS
 
-```jsx
-// with npm
-npm install @material-ui/system
+Often we find ourselves jumping from the JS to the CSS files, or from the components definition to the instance usage in order to conclude where and how some styles are defined. This is particularly true as the complexity (LOCs/# of elements) of the component we are working on increases. We could save a lot of time by removing this contraint.
```

Comment:
```suggestion
Often we find ourselves jumping from the JS to CSS, or from a component definition to an instance in order to understand where and how some styles are defined. This is particularly true as the complexity (LOCs/# of elements) of the component we are working on increases. We could save a lot of time by removing this constraint.
```

Reply:
In addition to Matt's suggestion, let's reduce the jargon
```suggestion
Often we find ourselves jumping from the JS to the CSS files, or from the components definition to the instance usage in order to conclude where and how some styles are defined. This is particularly true as the complexity (lines of code/number of elements) of the component we are working on increases. We could save a lot of time by removing this contraint.
```